po/reuse-docs.pot

```diff
@@ -0,0 +1,1858 @@
+# SOME DESCRIPTIVE TITLE
+# Copyright (C) YEAR Free Software Foundation, Inc.
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"POT-Creation-Date: 2020-04-27 22:22+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#. type: Plain text
+#: CHANGELOG.md
+#, markdown-text, no-wrap
+msgid ""
+"<!--\n"
+"SPDX-FileCopyrightText: 2019 Free Software Foundation Europe e.V.\n"
```

Comment:
Any way to exclude these strings from translation? They are the ones defining license and copyright, so should never be translated

Reply:
Not easily, it would require a custom pre-processing script.  It can easily be marked as something not to translate in Weblate.